src/CNLOADS.CPP

```diff
@@ -6580,7 +6581,9 @@ RC XSURF::xs_ASHWAT()		// subhour calcs for ASHWAT fenestration
 	{	// shades closed
 		xs_pFENAW[ 1]->fa_Subhr( min( fSC, 1.f), bDoFrm);
 	}
+#ifdef _WIN32
```

Comment:
Why do this?  Why not conditionally #define FPCHECK as needed for different platforms.  E.g., #define it as nothing if suitable functionality cannot be found.  Then you touch cnglob.h only, not every place FPCHECK is used.